compiler/damlc/daml-doc/src/DA/Daml/Doc/Annotate.hs

```diff
@@ -53,9 +53,10 @@ applyHide = concatMap onModule
         -- as that would be a security risk
         onTemplate x = [x | not $ isHide $ td_descr x]
         onFunction x = [x | not $ isHide $ fct_descr x]
+        onMethod x = [x | not $ isHide $ cm_descr x]
         onClass x
             | isHide $ cl_descr x = []
-            | ClassDoc {..} <- x = [x { cl_functions = concatMap onFunction cl_functions }]
+            | ClassDoc {..} <- x = [x { cl_methods = concatMap onMethod cl_methods }]
```

Comment:
It would be nice to move renaming to a separate PR in the future. This is already a very large diff.

Reply:
They have different types that contain different data, it's not just renaming.

Reply:
Fair enough.

Reply:
Apologies for the large PR. It is somewhat atomic, however.

Reply:
No worries, some PRs aren’t easily splittable and that’s totally fine :slightly_smiling_face: